doc/guides/device_mgmt/smp_groups/smp_group_0.rst

```diff
@@ -406,8 +406,13 @@ where:
 System reset
 ************
 
-Performs reset of system. The device should issue response before resetting so that
-the SMP client could receive information that the command has been accepted.
+Performs reset of system. The device should issue response before resetting so
+that the SMP client could receive information that the command has been
+accepted. By default, this command is accepted in all conditions, however if the
+Kconfig option `OS_MGMT_RESET_HOOK` is enabled and an application registers a
```

Comment:
Use the :kconfig:option:`CONFIG....` tag.

Reply:
Done